flink-core/src/main/java/org/apache/flink/configuration/QueryableStateOptions.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.configuration;
+
+import org.apache.flink.annotation.PublicEvolving;
+
+import static org.apache.flink.configuration.ConfigOptions.key;
+
+/**
+ * The set of configuration options relating to Queryable State.
+ */
+@PublicEvolving
+public class QueryableStateOptions {
+
+	// ------------------------------------------------------------------------
+	// Server Options
+	// ------------------------------------------------------------------------
+
+	/** Flag to indicate whether to start the queryable state server. */
+	public static final ConfigOption<Boolean> START_SERVER =
+			key("query.server.start").defaultValue(true);
```

Comment:
I other places, we put the default value on the next line. Assuming more aspects to come (deprecated keys, documentation text), this makes it more readable.